pages/your_problem/data_licence.md

```diff
@@ -5,36 +5,49 @@ contributors: [Siiri Fuchs, Minna Ahokas]
 search: exclude
 ---
 
-## "Concrete Problem 1, Ideally Formulated as a Question"
+## "Why should I assign a licence to my research data?"
  
-### Description
-
-Short explanation of what this problem is.
+### Description 
+Licensing is an important aspect of meeting the R (reusable) principle in FAIR data management. As part of the publication process, you need to decide under which license the data is released. A license is like a greeting to any person wishing to access, cite and use your data telling what they are allowed to do with the data.
 
 ### Considerations
-
-* Optional field, ideally structured in bullet points, detailing things to consider about this problem
+* Remember that a licence is needed also if you use someone else’s data, that is protected by copyright or related rights.
+* If no license is granted, re-use is generally not permitted without the consent of the copyright holder.
+* Be sure of data ownership before publishing data. 
+  * Are there rights belonging to a third party?
 
 ### Solutions
-* Bullet point solution 1
-  * Sub-point
-* Bullet point solution 2
+* Make your research data available under an appropriate licence, which defines the degree of publicity and rights to use your data.
+* Choose a licence that ensures your data is attributed to you correctly and makes the terms of re-using your data explicit to the user.
+
+## Related topics
+* [Sharing] (sharing)
+
+
 
-## "Concrete Problem 2, Ideally Formulated as a Question"
+
+## "What licence should I apply to my research data?"
  
-### Description 
-Same as above
+### Description
+
+This depends on what rights protect your research data. Which licence to choose might be governed by university policy or funders’ mandates. Research data can have varying degrees of publicity. There are circumstances in which data may be subject to restrictions eg. if datasets contain sensitive information. 
 
 ### Considerations
-Same as above
 
-### Solutions
-Same as above
+* If possible, choose and apply the least restrictive licence possible to ensure the widest possible re-use.
+* Remember that if you publish your data in a data repository of your choice, a licence agreement will be applied to your data.
+  * Repositories can be selected based on data licence and sharing policy by using [re3data.org] (https://www.re3data.org/)
+* ELIXIR data resources ideally have terms of use or a licence that enables the reuse and remixing of data.
 
-## Related topics
-(Optional section)
-* Bullet point list of other pages in this website that are connected to this lifecycle stage
+### Solutions
+* Apply a Creative Commons license to your data to ensure that your data can be shared and reused. 
+  * CC licences are conformant with the set forth in the [Open Definition](https://opendefinition.org/licenses/) and they are available in human-readable and machine-readable forms and they offer different levels of permission. 
```

Comment:
```suggestion
* Apply to your data one of the [recommended licenses conformant to the Open Definition](https://opendefinition.org/licenses/), so that your data can be shared and reused.
  * The [Open Definition](https://opendefinition.org/) sets out principles that define the meaning of "open" in relation to data and content.
  * [Creative Commons licenses](https://creativecommons.org/licenses/) are the best known open data licenses and are available in human-readable and machine-readable forms, with different levels of permissions.
```

Reply:
Thanks for the comments & changes - much appreciated!